src/hoverifier.test.ts

```diff
@@ -34,7 +34,7 @@ describe('Hoverifier', () => {
             scheduler.run(({ cold, expectObservable }) => {
                 const hoverifier = createHoverifier({
                     closeButtonClicks: new Observable<MouseEvent>(),
-                    goToDefinitionClicks: new Observable<MouseEvent>(),
+                    goToDefinitionClicks: new Subject<MouseEvent>(),
```

Comment:
Why this change?

Reply:
To enable this (can't `next` an observable): https://github.com/sourcegraph/codeintellify/pull/42/files#diff-8a128e9e8a5a8bb9767f5f5392391217R434

Reply:
👍 btw you can just pass the subject in to subscribe instead of wrapping it in an arrow func and calling next on it